rhel-3tier-iaas/README.md

```diff
@@ -0,0 +1,56 @@
+# Red Hat Linux 3-tier IaaS Solution on Azure 
+## Solution Overview 
+This Azure Quick Start template deploys a 3 Tier Red Hat Solution on Azure.The Solution includes Web tier Servers, Application tier Servers and Database Tier Servers running Red Hat Enterprise Linux 7.3. Template follows Standard best practices for running a 3 tier Red Hat Linux IaaS workload on Azure. This template will deploy multiple VMs in each tier as per requirement. 
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Frhel-3tier-iaas%2Fazuredeploy.json" target="_blank">
+<img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.png"/>
+</a>
+<a href="http://armviz.io/#/?load=https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Frhel-3tier-iaas%2Fazuredeploy.json" target="_blank">
+<img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.png"/>
+</a> 
+
+ 
+##Deployment Solution Architecture 
+
+This template will deploy: 
+
+- Four storage accounts: 3 for storing VM's of each tier, 1 for storing diagnostics data.
+-	One Virtual Network with four subnets
+- 4 Network Security Groups, one for each subnet
+-	External Load Balancer to load balance Web Traffic(HTTP & HTTPS) to web servers
+- Internal Load Balancer to load balance traffic for app VM's
+-	2 Public IP’s, one for external Load balancer and other for Jump VM. 
+-	3 Virtual Machine Availability sets for Web Tier, Application Tier and Database tier.
+-	One Jump VM to faclitate ssh access to all other VMs
+-	Multiple Red Hat Enterprise Linux VMs as per parameter value given during deployment for each tier
+
+![Deployment Solution Architecture](https://github.com/Azure/azure-quickstart-templates/master/redhat-3tier-architecture/images/architecture.png?raw=true)
+
+##Licenses and Costs 
+
+The Red Hat Enterprise Linux 7.3 image used in this solution is the PAYG model and doesn't require the user to license it, it will be licensed automatically after the instance is launched first time. Use of this image carries a separate hourly charge that is in addition to Microsoft's Linux VM rates. Total price of the VM consists of the base Linux VM price plus RHEL VM image surcharge.  Click [here](https://azure.microsoft.com/en-us/pricing/details/virtual-machines/red-hat/) for pricing details.
+
+##Prerequisites 
+
+- Azure Subscription with specified payment method (Red Hat Enterprise Linux is a market place product and requires payment method to be specified in Azure Subscription
+-	User account with admin or contributor access to the given azure subscription
```

Comment:
Is this prereq needed only for template deployment or something else?  (all samples in the repo need contributor access)